test/scaleTest.js

```diff
@@ -0,0 +1,39 @@
+/* global suite test assert */
+
+function isAnimationEqual (keyframesA, keyframesB, currentTime) {
+  var timing = {duration: 2000, iterations: 5, direction: 'alternate', easing: 'linear'};
+  var targetA = document.createElement('div');
+  document.body.appendChild(targetA);
+
+  var targetB = document.createElement('div');
+  document.body.appendChild(targetB);
+
+  var animation = targetA.animate(keyframesA, timing);
+  animation.currentTime = currentTime;
+  var result = window.getComputedStyle(targetA).transform;
+
+  animation = targetB.animate(keyframesB, timing);
+  animation.currentTime = currentTime;
+  var expected = window.getComputedStyle(targetB).transform;
+
+  targetA.parentNode.removeChild(targetA);
+  targetB.parentNode.removeChild(targetB);
+
+  return result === expected;
+}
```

Comment:
I would still like to see a check in this function that the expectation transform values provided are valid.